### PR TITLE
Update reception process for SR2023

### PR DESCRIPTION
**Pull request by sedders123**

### CODE REVIEW

File: docs/competition/teams/role-descriptions/reception.md [hunk 1]
```diff
@@ -56,11 +47,9 @@ Volunteers will be turning up throughout the competition, it's important to make
 
 You should have the following items present for your duties. If you do not, please let the competition event coordinator know as soon as possible.
 
-* Spare Media Consent Forms (MCFs) to allow visitors to enter.
-* Pens to fill in forms.
-* Wristbands to give to entrants.
+* Pens.
 * Competitor booklets to give to teams.
```
Comment (PeterJCLaw) [inline docs/competition/teams/role-descriptions/reception.md:51]: The flow chart doesn't mention handing out swag -- should it?

File: docs/competition/teams/role-descriptions/reception.md [hunk 2]
```diff
@@ -56,11 +47,9 @@ Volunteers will be turning up throughout the competition, it's important to make
...
-* This documentation and flow chart.
+* This documentation.
 * Sign-in/out sheet.
```
Comment (PeterJCLaw) [inline docs/competition/teams/role-descriptions/reception.md:53]: Is this the same sheet for teams & volunteers or are they separate? Maybe clarify this either way?

File: docs/competition/teams/role-descriptions/reception.md [hunk 3]
```diff
@@ -56,11 +47,9 @@ Volunteers will be turning up throughout the competition, it's important to make
 
 You should have the following items present for your duties. If you do not, please let the competition event coordinator know as soon as possible.
 
-* Spare Media Consent Forms (MCFs) to allow visitors to enter.
-* Pens to fill in forms.
-* Wristbands to give to entrants.
+* Pens.
 * Competitor booklets to give to teams.
-* This documentation and flow chart.
+* This documentation.
```
Comment (PeterJCLaw) [inline docs/competition/teams/role-descriptions/reception.md:52]: Also a copy of the safeguarding policy, plus whatever is needed to record that volunteers have signed it?

Related to that: how will reception know who has already signed it ahead of time?

File: docs/competition/teams/role-descriptions/reception.md [hunk 4]
```diff
@@ -9,23 +9,15 @@ original:
...
 
 At the start of the competition day, there will be four reception staffers
 covering the entrance. Once the initial influx of people is complete, this
-will reduce. Additionally, after the initial influx, receptionists
-should allow people wearing SR wristbands to enter the building at will (they
-should check that people walking in have one, or follow the procedure above).
+will reduce.
 
-Throughout the competition, receptionists typically get a lot of questions by the public. Any member of the public should be allowed into the building if they sign a Media Consent Form. Feel free to answer, give them a booklet, and send them to Helpdesk (let them sign in) if they seem interested.
+Throughout the competition, receptionists typically get a lot of questions by the public. Any member of the public should be allowed into the building if they're happy photos and videos might be taken of them. Feel free to answer, give them a booklet, and send them to Helpdesk (let them sign in) if they seem interested.
```
Comment (PeterJCLaw) [inline docs/competition/teams/role-descriptions/reception.md:20]: This implies we're denying entry to members of the public who don't want to be in photos -- is that so? (If so, maybe make that explicit)

Comment (PeterJCLaw) [inline docs/competition/teams/role-descriptions/reception.md:20]: "Feel free to answer" feels clunky here, I'm not sure what it's trying to say.

Comment (PeterJCLaw) [inline docs/competition/teams/role-descriptions/reception.md:20]: > send them to Helpdesk

What are the members of the public expected to do at Helpdesk? Is that just a place where they can ask more questions/be told more about what's going on? If so, it's not clear to me why reception can't handle that? (too)

**Review by PeterJCLaw — COMMENTED** [review]
Thanks for picking this up.
Generally :+1:, some questions & suggestions inline.

**Review by antoinepetty — APPROVED** [review]
Great work but Peter's changes would help too

**Review by Adimote — APPROVED** [review]

File: docs/competition/teams/role-descriptions/reception.md (hunk 2 above)
Comment (sedders123) [reply docs/competition/teams/role-descriptions/reception.md]: These have been sepereated

File: docs/competition/teams/role-descriptions/reception.md (hunk 4 above)
Comment (sedders123) [reply docs/competition/teams/role-descriptions/reception.md]: Have tweaked this

**Review by PeterJCLaw — APPROVED** [review]
:+1:

Would still be great to work out how we're going to know who has already read & signed the safeguarding policy. Similarly if we want to ensure that volunteers have seen the risk assessment then should that be mentioned here?

**Pull request merged**